low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -185,8 +162,12 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup_init import SetupInit
+        try :
+            SetupInit(self.args).process()
+        except Exception as error:
+            print(error)
+            sys.exit(1)
```

Comment:
Is this expected, to handle print and exit in sspl_setup.py

Reply:
can also be called as  
SetupInit.validate(self.args)   => validate_args() that I have written in sspl_setup_init 
SetupInit.process()